runtime-interface/src/lib.rs

```diff
@@ -51,7 +58,7 @@ pub trait GearRI {
         wasm_mem_addr: Option<HostPointer>,
         wasm_mem_size_in_pages: u32,
         stack_end_page: Option<u32>,
-        program_prefix: Vec<u8>,
+        program_id: Vec<u8>,
```

Comment:
program id type

Reply:
unfortunately that requires `ProgramId` implementing `sp_runtime_interface::RIType`